alacritty_terminal/src/term/mod.rs

```diff
@@ -1586,7 +1591,30 @@ impl<T: EventListener> Handler for Term<T> {
 
     #[inline]
     fn move_forward_tabs(&mut self, count: u16) {
-        trace!("[unimplemented] Moving forward {} tabs", count);
+        trace!("Moving forward {} tabs", count);
+        self.damage_cursor();
```

Comment:
Is the damage cursor here necessary when we're damaging the entire line later anyway? We don't seem to be handling wrapped lines here, so I doubt it can jump across boundaries?

Reply:
I just did the same as other function.

Reply:
Well you probably added that to the other function when adding damage tracking, doesn't really mean it's necessarily correct? Seems worth investigating because logically it checks out to me.